README.md

```diff
@@ -195,7 +195,7 @@ Settings can be provided from the command line, a configuration file, or an envi
 | default-balances-limit        |         | default-balances-limit        | INDEXER_DEFAULT_BALANCES_LIMIT        |
 | max-applications-limit        |         | max-applications-limit        | INDEXER_MAX_APPLICATIONS_LIMIT        |
 | default-applications-limit    |         | default-applications-limit    | INDEXER_DEFAULT_APPLICATIONS_LIMIT    |
-
+| data-dir                      | i       | data-dir                      | INDEXER_DATA_DIR                      |
```

Comment:
Is this `INDEXER_DATA` possibly? Just wondering because of the change in `misc/Dockerfile`

Reply:
I think it's like this because we have a command line flag, an automatically generated env variable (with viper) and manually check for `INDEXER_DATA`.

We should make a viper alias instead of manually checking `INDEXER_DATA`.